apps/app/components/pages/single-page-block.tsx

```diff
@@ -36,6 +36,7 @@ import { PAGE_BLOCKS_LIST } from "constants/fetch-keys";
 
 type Props = {
   block: IPageBlock;
+  index: number;
```

Comment:
I don't see the need for this index prop, can we please remove it?

Reply:
sure